rust/crd/src/security.rs

```diff
@@ -66,6 +73,9 @@ impl KafkaTlsSecurity {
     pub const CLIENT_PORT: u16 = 9092;
     pub const SECURE_CLIENT_PORT_NAME: &'static str = "kafka-tls";
     pub const SECURE_CLIENT_PORT: u16 = 9093;
+    pub const BOOTSTRAP_PORT_NAME: &'static str = "bootstrap";
```

Comment:
There are now a lot of ports and it is not obvious for the user which one to use. The documentation should clarify this.

Reply:
See 109ad3c

Reply:
I thought you meant in the code, but did you actually mean the user docs?

Reply:
See c9336ed